ninja.lua

```diff
@@ -76,6 +90,8 @@ function ninja.generateSolution(sln)
 
 	p.w("# default target")
 	p.w("default " .. cfg_first)
```

Comment:
However, seems that some projects might not have cfg.platform (like premake it self).
So when I try to compile premake I get this:

Error: /home/rui/dev/premake/modules/ninja/ninja.lua:61: attempt to concatenate field 'platform' (a nil value), so I will do a nil check before trying to append it.